### PR TITLE
#5662 Add Vogtle 3 nuclear capacity

### PR DESCRIPTION
**Pull request by MisterClean**
## Issue
#5662 

## Description
Implements a new nuclear generation capacity total for US-SE-SOCO & US zones. On July 31 2023 Vogtle Unit 3 was commissoned, adding 1,114MW nameplate electrical capacity to the Southern Company grid.

🇺🇸 🦅 ⚛️
https://www.ajc.com/news/first-new-vogtle-nuclear-reactor-enters-operation-making-history/QUW3BTDRJRDT7MRWB3UHLRWX4A/

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [ ] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

**Comment by MisterClean** [comment]
@VIKTORVAV99 

> Do you have another source? It sees to be blocked in the EU (due to GDPR).
In the [June 2023](https://www.eia.gov/electricity/data/eia860m/) EIA Form 860M look for Entity ID 7140, Generator ID 3. EIA has nameplate electric at 1,114MW for this unit.
image

> We also need to update the DATA_SOURCES.md file with the new source for nuclear unless the current source is already up to date. (doubtful).
Just added this!

> Also do you think it would make sense to create a script to update all the capacities for US zones using the EIA dataset linked in issue?
I don't know enough about this data set to make that judgment. We could miss behind-the-meter generation if we do it that way.

**Comment by VIKTORVAV99** [comment]
> @VIKTORVAV99
> 
> > Do you have another source? It sees to be blocked in the EU (due to GDPR).
> 
> In the [June 2023](https://www.eia.gov/electricity/data/eia860m/) EIA Form 860M look for Entity ID 7140, Generator ID 3. EIA has nameplate electric at 1,114MW for this unit. World Nuclear News also has [an article](https://www.world-nuclear-news.org/Articles/Vogtle-3-begins-commercial-operation) about the commercial service starting. image
> 
> > We also need to update the DATA_SOURCES.md file with the new source for nuclear unless the current source is already up to date. (doubtful).
> 
> Just added this!
> 
Thanks!

> > Also do you think it would make sense to create a script to update all the capacities for US zones using the EIA dataset linked in issue?
> 
> I don't know enough about this data set to make that judgment. We could miss behind-the-meter generation if we do it that way.

Hmm, agreed. Should be safe to use for everything except solar though (and possibly wind). I doubt there is any behind the meter production for the rest that would be reported. The reason I ask is that I have been exploring and looking for sources that could be used for automatic capacity parsing as that is something I think would benefit everyone involved.

File: DATA_SOURCES.md [hunk 1]
```diff
@@ -431,7 +431,9 @@ For many European countries, data is available from [ENTSO-E](https://transparen
 - United States of America
   - Federal: [EIA](https://www.eia.gov/electricity/data.cfm#gencapacity)
   - Balancing Authorities: [EIA](https://www.eia.gov/electricity/data/eia860M/)
-  - States: [EIA](https://www.eia.gov/electricity/data/state/)
+  - States:
+    - [EIA](https://www.eia.gov/electricity/data/state/)
+    - [EIA](https://www.eia.gov/electricity/data/eia860M/)
```
Comment (VIKTORVAV99) [inline DATA_SOURCES.md:436]: Is this needed with the above link on the federal level?

Comment (MisterClean) [reply DATA_SOURCES.md:436]: yes that would probably help

Comment (MisterClean) [reply DATA_SOURCES.md:436]: actually, the data set is listed on that page

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

**Comment by MisterClean** [comment]
Thanks @VIKTORVAV99! Super stoked on Vogtle!!

**Comment by VIKTORVAV99** [comment]
> Thanks @VIKTORVAV99! Super stoked on Vogtle!!

Yeah! While I'm a little conflicted about nuclear there is no doubt that it's critical to the decarbonization of the grid in places where hydro can't supply all of the baseload.

**Comment by MisterClean** [comment]
image

**Pull request merged**